src/components/Book.js

```diff
@@ -1,11 +1,18 @@
 /* eslint-disable react/prop-types */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉